agentstack/generation/task_generation.py

```diff
@@ -87,3 +87,8 @@ def generate_crew_task(
     ]
 
     insert_after_tasks(file_path, code_to_insert)
+
+
+def get_task_names(framework: str, path: str = '') -> List[str]:
```

Comment:
Should have a conditional for the framework.

Reply:
Oh, wait, that does get passed down. `get_crew_components` should be crew-specific.

Reply:
so this is kind of unclear at the moment.

Crew is a very opinionated framework that has two components: agents and tasks

Autogen is extremely unopinionated and isnt as simple as to have design boil down to agents/tasks. I can explain the pros and cons later, but my assumption is when we implment autogen/AG2 next, we'll need to enforce _some_ opinions of our own in order to make it possible to build a project with it using AgentStack. This very well may end up being agent/task components like Crew.

tl;dr: idk what other framework structures will look like yet, so i'm leaving this function ambiguous intentionally